src/app/GenerateTypes.ts

```diff
@@ -0,0 +1,96 @@
+/* Import Dependencies */
+import { writeFileSync } from 'fs'
```

Comment:
When is this triggered? Is this once when you added them or does it run on each build?

Reply:
I (now) added it to the Dockerfile, so it will refresh the types upon each build.
The script needs to be triggered via command as it is a server sided script.